__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data/index");
+const app = require("../app");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET/api/tpics", () => {
+    test("200: responds with an array of topic objects", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                expect(Array.isArray(body.topics)).toBe(true);
+                body.topics.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug", expect.any(String));
+                    expect(topic).toHaveProperty("description", expect.any(String));
+                });
+            });
+    });
+    test("404: responds with an error when the path is invalid", () => {
+        return request(app)
+            .get("/api/not-a-path")
```

Comment:
This test isn't specific to this endpoint. It should therefore be moved into it's own describe block for generic 404 errors 👍